bidscoin/plugins/dcm2niix2bids.py

```diff
@@ -308,6 +308,8 @@ def bidscoiner_plugin(session: Path, bidsmap: dict, bidsses: Path) -> Union[None
             if bcoin.run_command(command):
                 if not list(outfolder.glob(f"{bidsname}.*nii*")): continue
 
+            jsonfiles.update(outfolder.glob(f"{bidsname}.json"))  # add existing created json files: bidsname.json
```

Comment:
This line should also be added right after the physio2tsv conversion above (i.e. after line 295)